workflows/bacterial_genomics/bacterial_genome_annotation/bacterial_genome_annotation-tests.yml

```diff
@@ -54,9 +55,15 @@
       asserts:
         has_n_columns:
           n: 8
-    tooldistillator_summarize_annotation:
+    tooldistillator_summarize_annotation_without_bakta:
```

Comment:
Those outputs should eventually also follow the human readable recommendation but for an update this is fine.